shiny/render/_render.py

```diff
@@ -96,35 +101,75 @@ def text(
 # ======================================================================================
 # RenderPlot
 # ======================================================================================
+
+
+def _default_plot_ui(
+    id: str, *args: Any, _params: dict[str, Any] | None = None, **kwargs: Any
+) -> Tag:
+    # The _params argument contains the arguments that were passed to `@render.plot`. In
+    # the specific case of `@render.plot`, we grab the width and height arguments from
+    # the decorator and apply them to the output_plot call.
+    if _params is not None:
+        extra_args = {}
+
+        if "width" in _params:
+            extra_args["width"] = (
+                f"{_params['width']}px"
+                if isinstance(_params["width"], (int, float))
+                else str(_params["width"])
+            )
+
+        if "height" in _params:
+            extra_args["height"] = (
+                f"{_params['height']}px"
+                if isinstance(_params["height"], (int, float))
+                else str(_params["height"])
+            )
+
+        kwargs.update(extra_args)
```

Comment:
Reverse the order here, so that `kwargs` wins.

Reply:
I couldn't reverse the order without upsetting the type checker, but I filtered out the extra_args that would conflict.